include/deal.II/dofs/dof_tools.h

```diff
@@ -1697,6 +1697,82 @@ namespace DoFTools
   extract_locally_relevant_dofs(const DoFHandlerType &dof_handler,
                                 IndexSet &            dof_set);
 
+
+  /**
+   * Extract the set of local, active or relevant DoF indices for each
+   * components within the mask that are owned by the current  processor. For
+   * components disabled by the mask, an empty IndexSet is returned.
+   * For regular DoFHandler objects, this set is the complete set with all DoF
```

Comment:
what is a "regular" DoFHandler? I think you want to say "a DoFHandler built on a sequential triangulation".